### ISSUE TITLE
MacOS tensorflow ERROR

### ISSUE THREAD

**Issue by 7NT** [lorserker/ben#33]
OS: macOS Ventura 13.3.1 (a)
python: Python 3.10.10

During conda_setup.sh, got an ERROR:
```
ERROR: Could not find a version that satisfies the requirement tensorflow==1.15 (from versions: 2.12.0rc0, 2.12.0rc1, 2.12.0)
ERROR: No matching distribution found for tensorflow==1.15
```

Do I need to uninstall python 3.10 then install 3.7?
Will it run in tensorflow 2?

Thanks

**Comment by 7NT** [comment]
later, I bypassed this error with (changed to /src folder):
```
conda install -c conda-forge websockets
conda install -c conda-forge numpy
conda install -c conda-forge tensorflow==1.15
```

**Comment by haliteris** [comment]
I always found the conda installer as the safest option, as not to cause any mismatch-version problems. Works for both Tensorflow 1 and 2, CPU and GPU.

**Comment by lorserker** [comment]
@haliteris thanks for sharing your solution!

**Comment by ThorvaldAagaard** [comment]
As this is solved I think the issue should be closed.

**Comment by lorserker** [comment]
i keep it for mac users to see it.
say a mac user runs into trouble, then they want to create an issue, but they see that there is a solution.
if i close it then they might not find the solution by themselves

**Comment by ThorvaldAagaard** [comment]
Then it is probably better to add it in a readme-file. But it is your call - Number of open issues often tells about the project :-)

**Comment by drk1806** [comment]
https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch, https://mirrors.ustc.edu.cn/anaconda/pkgs/main, https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/, https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/peterjc123/, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/, https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/, https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/msys2/, https://mirrors.ustc.edu.cn/anaconda/pkgs/main/, https://mirrors.ustc.edu.cn/anaconda/pkgs/free/, defaults

**Comment by drk1806** [comment]
I am not finding tensorflow anywhere

**Comment by ZiggerZZ** [comment]
This is fixed since ben is upgraded to Python 3.9+ and Tensorflow 2.0+ #58

**Comment by ThorvaldAagaard** [comment]
Agree
